### DISCUSSION TITLE
Printing asset labels.  Vendor recommendations.  Label printers

### DISCUSSION THREAD

**Discussion by LCCMactech** [snipe/snipe-it#9283] [Q&A] [answered]
Greetings.
Our team is close to going live with Snipe-IT and are in the process of researching printing vendors for tamper-resistant, scratch-resistant polyvinyl style labels.   
I have read the tested/suggested devices in the [Snipe-IT manual.    ](https://snipe-it.readme.io/docs/asset-labels)
Our first checkout deployment run will be around 500 assets.  Looking to have someone else print them, but we will need to occasionally do onesies-twosies label printing, so We will also be looking at label maker print devices and tape.  
Recommendations are appreciated.  Label maker Print devices will be interfaced with MacOS and Windows 10 computers.  
Thank you

**Comment by AlexanderWPapyrus** [comment, marked as answer]
Hi, 
we printed about 900 by now. The "Brother p-950 nw" works fine for us. (Extra Strength Adhesive Labels)
As it has Ethernet also the whole IT team can print.
regards,
Alex

**Reply by LCCMactech** [reply]
Alex, thank you for the reply and information. Do you use the P-Touch software?  IMO, it's pretty limited in functionality.

**Reply by AlexanderWPapyrus** [reply]
Hi, no not at all. We only set the lable settings we wanna have (sorry for German)

Only issue we have (but this is a snipe-it bug). We need to set the lable lengh over 2 inch, generate the lables and then set it back to 1.6 inch for printing.  Otherwise the Barcode will be unreadable cause the generated barcode.png resolution is relative to the lengh of the lable. The QR code is always fine.
![2021-03-15 09_22_14-Printing asset labels  Vendor recommendations  Label printers · Discussion #9283](https://user-images.githubusercontent.com/80526133/111123802-0e703c80-8570-11eb-8884-e1d597b8e354.png)
![2021-03-15 09_23_32-Update Label Settings __ Papyrus Asset management](https://user-images.githubusercontent.com/80526133/111124061-542d0500-8570-11eb-9282-2ac56cbf4a9f.png)
![2021-03-15 09_24_47-Labels](https://user-images.githubusercontent.com/80526133/111124069-5727f580-8570-11eb-9e74-01b5f1b78159.png)

Then you can print directly out of your browser by generating lables and printing them.

**Reply by LCCMactech** [reply]
Thank you!
